docs/source/datasets.rst

```diff
@@ -18,6 +18,12 @@ General use cases are as follows: ::
     for label, line in train_iter:
         tokens += tokenize(label, line)
 
+    # Starting iterator at some offset
+    train_iter = IMDB(split='train', offset={'train':1000})
```

Comment:
I think it might be more straightforward to only accept a single int for offset for now and broadcast it across all splits.

Reply:
But different splits potentially contains different number of data-points?  Perhaps we could simplify by accepting either tuple (in case split is tuple, we can do one to one mapping) or int if user wants to apply same offset to all the splits?

Reply:
Sounds good to me